includes/discovery/sensors/temperature/timos.inc.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+if ($device['os'] === 'timos') {
```

Comment:
This is another one of those where its in so many things and in the examples, its hard to know what is necessary and what isn't.